modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -110,6 +110,12 @@ Ciphers and the minimum TLS version of the configured security profile are refle
 * `Strict`: does not allow routes to claim the same host name across namespaces.
 * `InterNamespaceAllowed`: allows routes to claim different paths of the same host name across namespaces.
 
+`wildcardPolicy` describes how routes with wildcard policies should be handled for the Ingress Controller. `wildcardPolicy` controls use of routes exposed by the Ingress Controller based on the route's wildcard policy.
```

Comment:
The API appeared to have `wildcardPolicy` nested under `routeAdmission`, so I included the details in the table accordingly. I can bring them up if needed.

Reply:
That's correct, wildcardPolicy is under routeAdmission.

Reply:
s/should be handled for the Ingress Controller/should be handled by the Ingress Controller/

I know "for" is used in the API docs, but that needs to change.

Reply:
I would scrap this sentence:

```
`wildcardPolicy` controls use of routes exposed by the Ingress Controller based on the route's wildcard policy.
```

I think the first sentence does the job of explaining the field.